DQM/SiStripMonitorClient/src/SiStripDcsInfo.cc

```diff
@@ -296,6 +296,14 @@ void SiStripDcsInfo::fillStatus(){
     else fraction = 1 - faulty_det/total_det;
     DcsFraction_->Reset();
     DcsFraction_->Fill(fraction);
+    IsLumiGoodDcs_ = fraction > MinAcceptableDcsDetFrac_;
+    if(IsLumiGoodDcs_) nGoodDcsLumi_++;
+    for (std::map<std::string,SubDetMEs>::iterator it = SubDetMEsMap.begin(); it != SubDetMEsMap.end(); it++) {
+      for (std::vector<uint32_t>::iterator ifaulty = it->second.FaultyDetectors.begin(); ifaulty != it->second.FaultyDetectors.end(); ifaulty++) {
+        uint32_t detId_faulty = (*ifaulty);
+        if(IsLumiGoodDcs_) it->second.NLumiDetectorIsFaulty[detId_faulty]++;
```

Comment:
I would think that it->second.NLumiDetectorIsFaulty[detId_faulty] needs to be initialized to 0 if not already in the map?
@ASPatterson